packages/src/lib/navigation/navigation.component.html

```diff
@@ -1,16 +1,21 @@
-<div class="navigation-header" [class.--is-handset]="isHandset()">
-  <div class="container">
-    <div class="row">
-      <div
-        #tabsContainer
-        class="navbar-tab col align-self-end"
-        [class.order-2]="isHandset() && hasOverflow()"
-      >
-        <nav mat-tab-nav-bar [tabPanel]="tabPanel" [mat-stretch-tabs]="false">
-          @for (link of linksFiltered(); track link) {
+<div class="sdg-navigation-header">
+  <div class="sdg-navigation-header-container" [class]="headerContainerClass()">
+    <div
+      #headerRow
+      class="sdg-navigation-header-row"
+      [class.--two-lines]="hasActions() && hasHandsetOverflow()"
+    >
+      <div #tabsSection class="sdg-navigation-tab-panel">
+        <nav
+          mat-tab-nav-bar
+          [tabPanel]="tabPanel"
+          [mat-stretch-tabs]="false"
+          disablePagination
+        >
+          @for (link of linksInTabs(); track link) {
             <a
               mat-tab-link
-              class="nav-link"
+              class="sdg-navigation-tab-link"
               [routerLink]="link.path"
               routerLinkActive="active"
```

Comment:
Pour la classe `active`, on devrait peut-être mettre `--active` comme les autres composants?. Il y a d'autres composants à `--is-active` aussi qu'on pourrait unifier éventuellement.